tests/runtest.sh

```diff
@@ -733,8 +733,10 @@ function run_test {
 # Variables for running tests in the background
 if [ `uname` = "NetBSD" ]; then
     NumProc=$(getconf NPROCESSORS_ONLN)
-else
+elif [ `uname` = "Darwin" ]
```

Comment:
The CI tests are failing due to the missing `; then` after the condition

Reply:
Thank you for catching this. I added the change to build.sh as well.